tango.py

```diff
@@ -209,7 +209,7 @@ def getInfo(self):
     #
     # Helper functions
     #
-    def resetTango(self, vmms):
+    def resetTango(self, vmms: list) -> None:
```

Comment:
`list` should be `List[<some type]` instead, and can be imported with `from typing import List`